x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectAuthenticator.java

```diff
@@ -288,6 +289,7 @@ private void getUserClaims(
                         })
                     );
             } else {
+                LOGGER.debug("ID Token header: {}", idToken.getHeader());
```

Comment:
Should we log idToken.getJWTClaimsSet() too?

Reply:
I was trying to keep the scope limited. But since JWT realm does something similar already, I think we can do it here too.